openpype/settings/defaults/project_settings/maya.json

```diff
@@ -337,6 +337,64 @@
             "families": [
                 "rig"
             ]
+        },
+        "ValidatePathForPlugin": {
+            "enabled": true,
+            "optional": false,
+            "active": true,
+            "attribute": {
+            "AlembicNode": "abc_File",
```

Comment:
If possible it would be good to indicate to the user that its the node type and node attribute needed. If not possible in the settings view, then maybe in the documentation.

Reply:
I think we can add some comments here but it definitely needs more detail documentation as the user's guide